widgets/social-media-buttons/social-media-buttons.php

```diff
@@ -143,6 +143,17 @@ function get_widget_form(){
 							'justify' => __( 'Justify', 'so-widgets-bundle' ),
 						),
 					),
+					'mobile_align'      => array(
+						'type'    => 'select',
+						'label'   => __( 'Mobile Alignment', 'so-widgets-bundle' ),
```

Comment:
The field above this one is labelled 'Align', so for consistency this should be 'Mobile align'.